### PR TITLE
Advanced configuration is moved to sidebar

### PR DESCRIPTION
**Pull request by BondarenkoStas**
Changed docs as advanced configuration is moved to sidebar and not in beta anymore

### CODE REVIEW

File: content/building/yaml.md [hunk 1]
```diff
@@ -23,18 +23,17 @@ The YAML feature is currently in *beta* and has the following limitations:
...
+Note that in order to use the file for build configuration on Codemagic, it has to be committed to your repository. The name of the file must be `codemagic.yaml` and it must be located in the root directory of repository.
 
 ## Encrypting sensitive data
 
 During the export, Codemagic automatically encrypts the secret environment variables in your build configuration. 
 
 If you wish to add new environment variables to the YAML file, you can encrypt them via Codemagic UI. 
 
-1. In your app settings > Advanced configuration (beta), click **Encrypt environment variables**.
+1. In your app settings on the right sidebar in the **Configuration as code** section, click **Encrypt environment variables**.
```
Comment (rimanteval) [inline content/building/yaml.md]: this on the right sidebar in the...  @helinanever sounds a bit rocky, maybe you have some rename ideas? or just put
app settings > right sidebar > configuration as code...?

File: content/building/yaml.md [hunk 2]
```diff
@@ -23,18 +23,17 @@ The YAML feature is currently in *beta* and has the following limitations:
 You can get started with YAML easily if you have an existing project set up on Codemagic. 
 
 1. Navigate to your app settings.
-2. Expand the **Advanced configuration (beta)** tab.
-3. Click **Download configuration** and save the generated `codemagic.yaml` file to a suitable location. 
+2. Click **Download configuration** on the right sidebar in the **Configuration as code** section.
 
-Note that in order to use the file for build configuration on Codemagic, it has to be committed to your repository. The name of the file must be `codemagic.yaml` and it must be located in the root directory of the project. 
+Note that in order to use the file for build configuration on Codemagic, it has to be committed to your repository. The name of the file must be `codemagic.yaml` and it must be located in the root directory of repository.
```
Comment (rimanteval) [inline content/building/yaml.md]: should this be _the_ repository instead of just repository?

File: content/building/yaml.md [hunk 3]
```diff
@@ -23,18 +23,17 @@ The YAML feature is currently in *beta* and has the following limitations:
...
 
 During the export, Codemagic automatically encrypts the secret environment variables in your build configuration. 
 
 If you wish to add new environment variables to the YAML file, you can encrypt them via Codemagic UI. 
 
-1. In your app settings > Advanced configuration (beta), click **Encrypt environment variables**.
+1. In your app settings on the right sidebar in the **Configuration as code** section, click **Encrypt environment variables**.
 2. Paste the value of the variable in the field or upload it as a file.
 3. Click **Encrypt**. 
 4. Copy the encrypted value and paste it to the configuration file.
```
Comment (rimanteval) [inline content/building/yaml.md]: Otherwise LGTM

File: content/building/yaml.md (hunk 1 above)
Comment (BondarenkoStas) [reply content/building/yaml.md]: Changed in a way it's written before https://github.com/codemagic-ci-cd/codemagic-docs/pull/55/commits/ed0d1ed35f16aae887ffe06b553b1a2d22704fc7

File: content/building/yaml.md (hunk 2 above)
Comment (BondarenkoStas) [reply content/building/yaml.md]: fixed in https://github.com/codemagic-ci-cd/codemagic-docs/pull/55/commits/ed0d1ed35f16aae887ffe06b553b1a2d22704fc7

**Pull request merged**